settings.php

```diff
@@ -240,6 +240,7 @@ function get_FTL_data($arg)
                                                 }
 
                                                 $FTLversion = exec("/usr/bin/pihole-FTL version");
+																								$FTLstart = exec("date -r /proc/" . $FTLpid . " '+%a %d %b %Y %R %Z'");
```

Comment:
Just a note:

The indentation is weird.
I think you used `tabs` to indent you code, but the rest of the code is using `spaces`.

Reply:
Fixed.

See: https://github.com/pi-hole/AdminLTE/issues/1983